tools/jvm.java.security

```diff
@@ -0,0 +1,2 @@
+networkaddress.cache.ttl=60
```

Comment:
Use an application-specific name for the file like `whois.java.security` so it's clear it's separate from the JVM